res/controllers/Midi_for_light-scripts.js

```diff
@@ -416,7 +416,7 @@ midi_for_light.deckVolumeChange = function(value, group, control) { // deck volu
         midi_for_light.deck_current = deckneu;
         midi.sendShortMsg(0x8F + midi_channel, 0x30, 0x64 + deckneu); // Note C on with 64 and add deck
         midi_for_light.volumeBeatBlockStatus = true;
-        midi_for_light.volumeBeatBlock_timer = engine.beginTimer(1000, "midi_for_light.volumeBeatBlock()");
+        midi_for_light.volumeBeatBlock_timer = engine.beginTimer(1000, function() { midi_for_light.volumeBeatBlock(); });
```

Comment:
Does this work, too? 

```suggestion
        midi_for_light.volumeBeatBlock_timer = engine.beginTimer(1000, midi_for_light.volumeBeatBlock);
```